firmware/openbot_nano/openbot_nano.ino

```diff
@@ -74,6 +76,7 @@ boolean coast_mode = 1;
 // HAS_BUMPER                           Enable/Disable bumper (1,0)
 // HAS_SPEED_SENSORS_FRONT              Enable/Disable front speed sensors (1,0)
 // HAS_SPEED_SENSORS_BACK               Enable/Disable back speed sensors (1,0)
+// HAS_SPEED_SENSORS_MIDDLE                Enable/Disable middle speed sensors (1,0)
```

Comment:
Indent should be the same as for other items.

Reply:
Fixed